templates/beta-post.adoc

```diff
@@ -0,0 +1,186 @@
+// // // // // // // //
+// ABOVE:
+// Update the second and third sentence appropriately for this release
+// // // // // // // //
+
+This Open Liberty beta introduces the following Jakarta EE 9 features which now possess their all-new Jakarta EE 9 package names:
+
+* ITEM_1
+* ITEM_2
+* ITEM_3
+// // // // // // // //
+// ABOVE:
+// Change the ITEMS above to list the new changes and features in this
+// beta release
+// // // // // // // //
+
+
+These join the Jakarta EE 9 features in link:https://openliberty.io/blog/?search=beta&key=tag[previous Open Liberty betas]:
+
+// // // // // // // //
+// Copy feature from both lists in the previous beta post creating    // a cumulative list from all previous betas to make a list of all    // existing features in the new beta.
+// // // // // // // //
+
+=== Try it now
+
+To try out these Jakarta EE 9 features on Open Liberty in a lightweight package, just update your build tools to pull the Open Liberty Jakarta EE 9 Beta Features package instead of the main release. The beta works with Java SE 14, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty.beta</groupId>
+    <artifactId>openliberty-jakartaee9</artifactId>
+    <version>20.0.0.RELEASE_VERSION-beta</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-jakartaee9', version: '[20.0.0.RELEASE_VERSION-beta,)'
+}
+----
+// // // // // // // //
+// Above:
+// Replace RELEASE_VERSION with the version number of Open Liberty
+// // // // // // // //
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+Enable the Jakarta EE 9 beta features in your app's `server.xml`. You can enable the individual features you want (but remember if you enable the `jsp-3.0` feature, the `servlet-5.0` and `el-4.0` features are automatically enabled for you) or you can just add the Jakarta EE 9 convenience feature to enable all of the Jakarta EE 9 beta features at once:
+
+[source, xml]
+----
+  <featureManager>
+    <feature>jakartaee-9.0</feature>
+  </featureManager>
+----
+
+Or you can add the Web Profile convenience feature to enable all of the Jakarta EE 9 Web Profile beta features at once:
+
+[source, xml]
+----
+  <featureManager>
+    <feature>webProfile-9.0</feature>
+  </featureManager>
+----
+
+=== HEADING
+
+// // // // // // // //
+// Remove if nothing else to say, or provide an update on aspect of the Jakarta EE 9 beta e.g- TCK
+// // // // // // // //
+
+[#allbeta]
+== All Beta Features package
+
```

Comment:
As above - switch the two sections round so that this package goes first but then also update the text as we did in the 20.0.0.10-beta post so that it makes sense if you read them in the new order.

Reply:
Done